template/src/app.test.ts

```diff
@@ -0,0 +1,16 @@
+import { screen } from "@testing-library/dom";
+import { start } from "tram-one";
+import app from "./app";
+
+describe("app", () => {
```

Comment:
why double quotes here?

Reply:
This is a side effect of the minimal prettier config in the project (which defaults to double quotes)